docs/contribute/doc.md

```diff
@@ -159,8 +159,10 @@ that declaration will become the body of the tactic doc entry. If there are
 multiple declarations, you can select the one to be used by passing a name to
 the `inherit_description_from` field.
 
-If you prefer a tactic to have a doc string that is different then the doc entry, then between
-the `/--` `-/` markers, write the desired doc string first, then `---` surrounded by new lines,
+If you prefer a tactic to have a doc string that is different than the doc entry,
```

Comment:
I think this is now a run-on sentence. But with @gebner 's change to use doc strings on the `add_tactic_doc` invocation itself, the `---` option isn't needed anymore. It was only there to allow the tactic doc string and the doc entry to be different, without forcing the use of a string literal. So I think we can remove reference to it entirely.

Reply:
If you can fix the docs, I'll make another PR soon-ish to fix the places where we use the `---` "feature."

Reply:
Fixed.